app/src/main/java/cm/aptoide/pt/home/HomeAnalytics.java

```diff
@@ -254,17 +273,39 @@ public void sendAppsChipHomeInteractEvent() {
   public void sendAppsChipInteractEvent() {
     final Map<String, Object> data = new HashMap<>();
     data.put(ACTION, APPS);
-    analyticsManager.logEvent(data, HOME_CHIP_CLICK, AnalyticsManager.Action.CLICK,
+    analyticsManager.logEvent(data, HOME_CHIP_INTERACT, AnalyticsManager.Action.CLICK,
         navigationTracker.getViewName(true));
   }
 
   public void sendGamesChipInteractEvent() {
     final Map<String, Object> data = new HashMap<>();
     data.put(ACTION, GAMES);
-    analyticsManager.logEvent(data, HOME_CHIP_CLICK, AnalyticsManager.Action.CLICK,
+    analyticsManager.logEvent(data, HOME_CHIP_INTERACT, AnalyticsManager.Action.CLICK,
         navigationTracker.getViewName(true));
   }
 
+  public void sendChipTapOnMore(String bundleTag, String chipTag) {
+    final Map<String, Object> data = createChipTapInteractMap(TAP_ON_MORE, bundleTag, chipTag);
+    analyticsManager.logEvent(data, HOME_CHIP_INTERACT, AnalyticsManager.Action.CLICK,
```

Comment:
Could just call the createChipTapInteractMap inside the log event, it is not necessary to store it on data